lib/chef/knife/rackspace_server_create.rb

```diff
@@ -459,7 +467,7 @@ def run
       else
         print "\n#{ui.color("Waiting for sshd", :magenta)}"
         print(".") until tcp_test_ssh(bootstrap_ip_address) {
-          sleep @initial_sleep_delay ||= 10
+          # sleep @initial_sleep_delay ||= 10
```

Comment:
If we don't remove this, and leave it commented, could we leave a comment as a breadcrumb for why? Just to provide later clue when someone comes across this and wonders "Why did they do this?"?